honeybee_radiance/cli/translate.py

```diff
@@ -149,6 +156,25 @@ def model_to_rad_folder(
     else:
         # re-serialize the Model and perform any checks
         model = Model.from_file(model_file)
+
+        if create_grids:
+            if not model.properties.radiance.has_sensor_grids:
+                sensor_grids = []
+                unit_conversion = {
+                    'Meters': 1,
+                    'Millimeters': 0.001,
+                    'Feet': 0.3048,
+                    'Inches': 0.0254,
+                    'Centimeters': 0.01
+                }
+                grid_size = 24 * 0.0254 / unit_conversion.get(model.units, 1)
+                offset = 30 * 0.0254 / unit_conversion.get(model.units, 1)
+                wall_offset = 12 * 0.0254 / unit_conversion.get(model.units, 1)
```

Comment:
I suggest using metrics and IP values here based on the user input. I know that you're using the IP distances as a base, but it will generate non-round values in SI and people won't be happy about that. I believe there are round number recommendations in SI that are close to the IP values.

Reply:
I added round values for SI units.